modules/identity-provider-oidc-supported.adoc

```diff
@@ -7,12 +7,13 @@
 
 The following OpenID Connect (OIDC) providers are tested and supported with {product-title}:
 
-* Active Directory Federation Services
+* Active Directory Federation Services for Windows Server
 +
 [NOTE]
 ====
-Currently, it is not supported to use Active Directory Federation Services with {product-title} when custom claims are used.
+Currently, it is not supported to use Active Directory Federation Services for Windows Server with {product-title} when custom claims are used.
 ====
+* Azure Active Directory
```

Comment:
Same two comments apply to this PR as https://github.com/openshift/openshift-docs/pull/47921